Assets/Mirror/Components/NetworkTransformBase.cs

```diff
@@ -39,6 +48,15 @@ public enum Compression { None, Much, Lots, NoRotation }; // easily understandab
         // This component could be on the player object or any object that has been assigned authority to this client.
         bool isClientWithAuthority => hasAuthority && clientAuthority;
 
+        [Tooltip("Changes to the transform must exceed these values to be transmitted on the network.")]
+        [SyncVar]
+        public Sensitivity sensitivity = new Sensitivity
```

Comment:
this seems overly complicated.
just add 3 public variables and avoid that struct magic please :)

Reply:
It's a struct because:
- it's a syncvar
- makes code intuitively readable
- looks nice in editor because it folds up
- could be formatted in the inspector via a property drawer to a horizontal 3-box layout
We use enums instead of check boxes too.

Reply:
https://en.wikipedia.org/wiki/KISS_principle - use the most simple solution for any given problem.
using a struct is the second most simple one :)

Reply:
done